Chapter1/Chapter1-1-1/Product.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace ReChapter1_1_1 {
+    /// <summary>
+    /// 商品クラス
+    /// </summary>
+    /// 
+    internal class Product {
+        /// <summary>
+        /// 商品コード
+        /// </summary>
+        public int Code { get; private set; }
+
+        /// <summary>
+        /// 商品名
+        /// </summary>
+        public string Name { get; private set; }
+
+        /// <summary>
+        /// 商品価格（税抜き）
+        /// </summary>
+        public int Price { get; private set; }
```

Comment:
【FYI: 情報共有。アクション不要です。】
こうしておくことで readonly なプロパティにできます。
コンストラクタで値を設定した後は変更されないことが明らかになるので、読むときに考慮することが減って読みやすくなります。
```suggestion
        /// <summary>
        /// 商品コード
        /// </summary>
        public int Code { get; }

        /// <summary>
        /// 商品名
        /// </summary>
        public string Name { get; }

        /// <summary>
        /// 商品価格（税抜き）
        /// </summary>
        public int Price { get; }
```